specs/spec-1-kudos.md

```diff
@@ -1,106 +1,310 @@
+Requires: n/a
 ---
 
-# RFP 2: Kudos
+# Spec 1: Kudos Widget
 
-# Summary
+<!-- Objective. In a few sentences, describe the key system objectives. Tweet length executive summary. -->
 
-Kudo is a widget to build a reputation using SBT tokens and a network effect. Kudo is a form of a recommendation to share a gratitude or achievement shared between humans.
+Kudos is a widget to build a reputation. The widget will allow any verified human to nominate, recieve, and interact with Kudos on Near Social.
 
-This widget will allow any verified human to give any other verified human a Kudo.
+## Changelog
+
+<!--
+Points for reviewers to consider:
+
+- Be pragmatic: Is there enough information to begin coding? Practically, implementation will reveal the unforeseeable. This doc can evolve to record those issues/resolutions.
+- Wide angle view: Do you know of overlapping projects? Are there OKRs or other non-technical forces that would push/pull priority?
+- Optimization: Describe alternate approaches and why they are better. Use Caveats section for approaches under consideration, even if they are ultimately abandoned. 
+
+--> 
+
+**Authors & Reviewers:** @starpause, @robert-zaremba, @htafolla
+*Please insert your name here if you do review the document.*
+
+| Ver. | Date       | Author          | Changes Made                             |
+| ------- | ---------- | --------------- | ---------------------------------------- |
+| 0.1.0   | 2023-04-21 | @starpause      | Initial draft of the document            |
+| 0.2.0   | 2023-04-23 | @starpause      | Merged prior writting by @robert-zaremba from his [Github version](https://github.com/near-ndc/gwg/pull/1)                   |
+| 0.3.0   | 2023-04-25 | @starpause      | Addressed review comments, formated sections to more closely match [template](https://github.com/near-ndc/gwg/blob/main/TEMPLATE-SPEC.md)                                                                            |
+
+# Open Questions
+
+<!-- Topics not covered in this document that need to be addressed in a meeting or async -->
+
+**Q:** Should use Social DB for upvotes?
+
+**A:**
+
+**Q:** Should we issue SBT when someone gives a Kudo (Con: Feels spammy, costs more)? Or when someone claims their SBT after reaching a # of upvotes (Con: Less SBT will be issued)?
+
+**A:**
+
+**Q:** What happens when a Kudo expires? 
+
+**A:**
+
+**Q:** Is commenting limited to verified humans?
+
+**A:**
+
+**Q:** Who would "follow a kudo" and why? What notifications would such a follow recieve? 
 
-## Context
+**A:**
 
-The goal of this widget is to foster and improve recognition and celebrations of individuals, nodes, projects, workgroups or collectives for their contribution to NEAR.
-This will server as a fundamental MVP for Reputation and Badges and create a general widget that can be forked and used for many use cases
+**Q:** What threshold can we define for when an audit is appropriate? 
 
-Other potential names for this widget:
+**A:**
+
+# Overview
```

Comment:
You can't have multiple header-1 (which is normally considered as a title) level sections . Not sure why this passes markdown linter.

Call me on telegram and I will explain a bit about the template

Reply:
Addressed the #, will try and catch you on TG!